app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,13 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
+    @user_posts = Post.where(author_id: @user.id)
+    # @posts_comments = Comment.where(user_id: )
```

Comment:
- [Required] I think it would be nice if you delete this piece of commented code from your code base, commented code makes your project look messy. If you need to review a previous version of your code, you can always use git history. Please remove this.